resources/articles/oauth/introspect.md

```diff
@@ -0,0 +1,79 @@
+:title POST /oauth/introspect
+
+:aside
+
+## See also
+
+* [POST /oauth/token](/oauth/token/)
+* [POST /oauth/ro](/oauth/ro/)
+* [GET /oauth/userinfo](/oauth/userinfo/)
+* [GET /oauth/jwks](/oauth/jwks/)
+* [GET /oauth/authorize](/oauth/authorize/)
+* [Server to Server Auth](/server-to-server-authentication/)
+* [Token introspection](/token-introspection/)
+
+:body
+
+The `/oauth/introspect` endpoint is used to verify token and to translate it into claims.
```

Comment:
"verify token and to translate it into claims" -> "verify a token and translate it to claims"